File: app/assets/javascripts/models/tactic.js
var app = app || {};

app.Tactic = Backbone.Model.extend({
  urlRoot: '/tactics',
  defaults: {
    team_id: null,
    formation: '4-4-2',
    tempo: 'balanced',
    playing_style: 'balanced',
    passing: 'balanced',
    tackling: 'balanced'
  }
});

File: app/models/tactic.rb
# == Schema Information
#
# Table name: tactics
#
#  id            :integer          not null, primary key
#  team_id       :integer
#  formation     :string           default("4-4-2")
#  tempo         :string           default("balanced")
#  playing_style :string           default("balanced")
#  passing       :string           default("balanced")
#  tackling      :string           default("balanced")
#  created_at    :datetime         not null
#  updated_at    :datetime         not null
#

class Tactic < ActiveRecord::Base
  has_one :team
end
